### PR TITLE
CCT-Chat Bot Test Cases Automation ~Chaudhryma

### PR DESCRIPTION
**Pull request by Mariachaudhry**

### CODE REVIEW

**Review by Mariachaudhry — COMMENTED** [review]
Reviewed

File: src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/Features/CCT_CHAT_BOT.feature
Comment (katm007) [inline src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/Features/CCT_CHAT_BOT.feature:1]: Please remove empty lines 73, 83

File: src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/StepImpl/CCT_CHAT_BOT_StepImpl.java
Comment (katm007) [inline src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/StepImpl/CCT_CHAT_BOT_StepImpl.java:1]: Please remove empty line 275

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
Please remove the empty lines from the feature file and Step Implementation class as Kate highlighted. thanks

File: src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/Features/CCT_CHAT_BOT.feature
Comment (matakevin) [reply src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/Features/CCT_CHAT_BOT.feature:1]: ^ I agree with Kate

File: src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/StepImpl/CCT_CHAT_BOT_StepImpl.java
Comment (matakevin) [reply src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/StepImpl/CCT_CHAT_BOT_StepImpl.java:1]: ^

**Review by matakevin — CHANGES_REQUESTED** [review]
Removed empty lines please

**Review by hamidnesar — APPROVED** [review]
Looks good!

**Review by katm007 — APPROVED** [review]
Approved!

File: src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/Features/CCT_CHAT_BOT.feature [hunk 1]
```diff
@@ -45,4 +45,46 @@ Feature: CCT CHAT BOT Scenarios
...
+  @SNOWPROJ-35 @Chaudhryma @Chaudhryma @Regression @playwright
+  Scenario: Test 1: Verify that the Global Research Training (D43) link appears
+    Given the user opens the Find Funding Chatbot on the NV
+    When selects Yes to Q1 regarding finding funding opportunities
+    And selects "Established Investigator" for Q2 regarding career stage
+    And selects "Outside the USA" for "Let's start with where do you plan to conduct your cancer research training?"
+    When the user selects "Yes" for Q5 "Do you consider yourself to belong to an underrepresented group in the biomedical research workforce?"
+    Then the following links should appear "Global Research Training (D43)"
+
+  @SNOWPROJ-35 @Chaudhryma @Chaudhryma @Regression @playwright
```
Comment (sarwarahmed1) [inline src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/Features/CCT_CHAT_BOT.feature:83]: Why is the Chaudryma tag used twice?

**Review by sarwarahmed1 — APPROVED** [review]

**Review by matakevin — APPROVED** [review]
LGTM

**Pull request merged**